playground/frontend/playground_components/pubspec.lock

```diff
@@ -0,0 +1,873 @@
+# Generated by pub
```

Comment:
Please delete this file. Libraries should not version-control `pubspec.lock`.
I will soon add it to `.gitignore`.